src/client/sandbox/code-instrumentation/properties/attributes-wrapper.js

```diff
@@ -2,6 +2,7 @@ import { isHammerheadAttr } from '../../../utils/dom';
 import fnBind from '../../../utils/fn-bind';
 import nativeMethods from '../../native-methods';
 import DomProcessor from '../../../../processing/dom';
+import { attrGetNamedItemInNonEnumerable } from '../../../utils/feature-detection';
```

Comment:
Do you mean this - `attrGetNamedItemIsNotEnumerable`?

Reply:
yep